scripts/GenerateCredentialSpecResource.ps1

```diff
@@ -37,19 +38,19 @@ Invoke-WebRequest "https://raw.githubusercontent.com/Microsoft/Virtualization-Do
 Import-Module $env:temp\cred.psm1
 
 # generate a unique docker cred spec name
-$dockerCredSpecName = "tmp-k8s-cred-spec" + -join ((48..57) + (97..122) | Get-Random -Count 64 | % {[char]$_})
+$dockerCredSpecName = "tmp-k8s-cred-spec" + -join ((48..57) + (97..122) | Get-Random -Count 64 | ForEach-Object {[char]$_})
 
 # have the upstream function perform its magic
 if (-not $Domain) {
     $Domain = Get-ADDomain
 }
-New-CredentialSpec -Name $dockerCredSpecName -AccountName $AccountName -Domain $Domain -AdditionalAccounts $AdditionalAccounts
+New-CredentialSpec -Name $dockerCredSpecName -AccountName $AccountName -Domain $Domain.DnsRoot -AdditionalAccounts $AdditionalAccounts > C:\gmsa\create-account.txt
```

Comment:
The redirection to the file is a pain if it starts erroring out say in CI. Is that needed? Wouldn't `Start-Transcript` above take care of that output too?

If really needed, how about try/catch-ing it, and cat-ing the file before throwing again?

Reply:
Sorry I had that in there while troubleshooting.  I have removed it.

Reply:
Thanks!